src/fu-util.c

```diff
@@ -3344,6 +3373,13 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Get all enabled plugins registered with the system"),
 		     fu_util_get_plugins);
+	fu_util_cmd_array_add(cmd_array,
+			      "download",
+			      /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+			      _("LOCATION"),
+			      /* TRANSLATORS: command description */
+			      _("Download a file using the client"),
```

Comment:
I don't think "using the client" is really needed.  Isn't that a detail that doesn't need to be exported "how" it's downloaded?

Reply:
True, fixed.